model/types.py

```diff
@@ -131,6 +128,12 @@ class OracleConfig(NamedTuple):
     price_threshold: int
     pairs: Set[Pair]
 
+
 class ImpactDelayConfig(NamedTuple):
     model: ImpactDelayType
     param_1: float
+
+
+reserve_balance_in_usd = float
+reserve_ratio = float
+floating_supply_stables_in_usd = float
```

Comment:
What's the intent here?

Reply:
left-over from an earlier implementation approach for the state variables